_docs_alerting/webhooks.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Webhooks
+description: Using webhooks to route alerts to other systems
+---
+
+Webhooks allow you to route CoScale alerts to other systems. You can use a webhook on an alert to route it to services that send messages to a team, or any other type of action.
+
+In order to use webhooks, follow the steps below:
+
+## Go to Alerts -> Manage
+
+Click on the "Create a notification scheme" button.
+
+## Click on "Add contact"
+
+Select "Webhook" as a contact method.
+
+<img alt="Add notification scheme buttons screenshot" src="{{ site.baseurl }}/gfx/alerting/notificationScheme/webhooks.png"/>
+
```

Comment:
Explain we have API and Webhook. Webhook being the simple system and API the more advanced.